hub-js/tsconfig.json

```diff
@@ -1,22 +1,9 @@
 {
+  "extends": "@tsconfig/recommended/tsconfig.json",
```

Comment:
Did you test that? Did it work on your localhost? Because I got:
```
(node:9335) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
/Users/mszostok/workspace/go/src/github.com/capactio/capact/hub-js/dist/src/index.js:1
import { ApolloServer } from "apollo-server-express";
^^^^^^

SyntaxError: Cannot use import statement outside a module
```

if I add `"type": "module"`, then:
```
> hub-js@0.1.0 start
> node --trace-warnings dist/src/index.js

node:internal/errors:464
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '<..>github.com/capactio/capact/hub-js/dist/src/schema' imported from <..>github.com/capactio/capact/hub-js/dist/src/index.js
```

I changed the ts config only because I was not able to make it work if the current one, so I decided to use the `recommeded`  one. I would appreciate information how to fix it e2e.

Reply:
You're right, build succeeded, but start didn't work :/

Here's working `tsconfig.json`:

```
{
  "compilerOptions": {
    "lib": ["es2020"],
    "module": "commonjs",
    "target": "es2020",
    "moduleResolution": "node",
    "allowSyntheticDefaultImports": true,
    "sourceMap": true,
    "strict": true,
    "noImplicitAny": true,
    "strictNullChecks": true,
    "noImplicitThis": true,
    "alwaysStrict": true,
    "noUnusedParameters": true,
    "noImplicitReturns": true,
    "noFallthroughCasesInSwitch": true,
    "esModuleInterop": true,
    "skipLibCheck": true,
    "outDir": "dist",
  },
  "include": ["src/**/*"],
  "exclude": ["node_modules"]
}
```

Make sure there is the following `type` in `package.json`:

```json
  "type": "commonjs"
```